app/src/main/java/com/cdio/solitaire/model/Card.kt

```diff
@@ -0,0 +1,46 @@
+package com.cdio.solitaire.model
+
+enum class Suit {
+    NA, CLUBS, DIAMONDS, HEARTS, SPADES; // Note the order: C = 1, D = 2, H = 3, S = 4.
+
+    fun short(): String {
+        return when (this.ordinal) {
+            1 -> "C"
+            2 -> "D"
+            3 -> "H"
+            4 -> "S"
+            else -> "NA"
+        }
+    }
+
+    fun isRed(): Boolean {
+        return when (this) {
+            DIAMONDS, HEARTS -> true
+            else -> false
+        }
+    }
+}
+
+enum class Rank {
+    NA, ACE, TWO, THREE, FOUR, FIVE, SIX, SEVEN, EIGHT, NINE, TEN, JACK, QUEEN, KING;
+
+    fun short(): String {
+        return when (this.ordinal) {
+            1 -> "A"
+            in 2..9 -> this.ordinal.toString()
+            10 -> "T"
+            11 -> "J"
+            12 -> "Q"
+            13 -> "K"
+            else -> "NA"
```

Comment:
Very clever use of ordinal values! 🙂 
However, I would argue it could be made more readable by also using direct enum values like so:
```suggestion
        return when (this) {
            ACE -> "A"
            TEN -> "T"
            JACK -> "J"
            QUEEN -> "Q"
            KING -> "K"
            NA -> "NA"
            else -> this.ordinal.toString()
```